GitUI/GitUIExtensions.cs

```diff
@@ -117,7 +117,11 @@ void OpenWithDiffTool()
                     && file.GetSubmoduleStatusAsync() is Task<GitSubmoduleStatus> task)
                 {
                     // Patch already evaluated
+#pragma warning disable VSTHRD103 // Call async methods when in an async method
+#pragma warning disable VSTHRD003 // Avoid awaiting foreign Tasks
```

Comment:
💡 Recommend filing follow-up issues for each of the VSTHRD suppressions (i.e. one issue covering all VSTHRD003, one issue covering all VSTHRD103, etc.).

Reply:
👍  I will raise a separate issue for each once this PR is merged.